addons/website_event/static/src/js/website_event_ticket_details.js

```diff
@@ -35,12 +35,16 @@ odoo.define('website_event.ticket_details', function (require) {
          * @private
          * @param {*} ev
          */
+         /* When the "Fold Tickets Details" option is active, this will be called each
+         time the user expand or fold the tickets (o_wevent_registration_btn). This
+         allows to show/hide elements depending on the folding state. */
         _onTicketDetailsClick: function (ev){
             ev.preventDefault();
             if (this.foldedByDefault){
-                $(ev.currentTarget).toggleClass('btn-primary text-left pl-0');
-                $(ev.currentTarget).siblings().toggleClass('d-none');
-                this.$('.close').toggleClass('d-none');
+                $(ev.currentTarget).toggleClass('btn-primary');
```

Comment:
Suggestion:

```js
let $target = $(ev.currentTarget);
$target.toggleClass('btn-primary');
$target.children().toggleClass('d-none');
$target.siblings('.o_wevent_registration_title, .o_price_range').toggleClass('d-none');
```

Creating a variable when you use the same call multiple (3+) times is usually easier for readability.
And since the "siblings" method takes a selector, using a coma separator should select everything you need in one line (please re-test to make sure it works though).